pkg/kubelet/images/image_gc_manager.go

```diff
@@ -248,28 +269,36 @@ func (im *realImageGCManager) detectImages(ctx context.Context, detectTime time.
 	im.imageRecordsLock.Lock()
 	defer im.imageRecordsLock.Unlock()
 	for _, image := range images {
-		klog.V(5).InfoS("Adding image ID to currentImages", "imageID", image.ID)
-		currentImages.Insert(image.ID)
+		imageKey := image.ID
+		if !isRuntimeClassInImageCriAPIEnabled {
+			klog.V(5).InfoS("Adding image ID to currentImages", "imageID", imageKey)
+		} else {
+			imageKey = getImageTuple(image.ID, image.Spec.RuntimeHandler)
+			klog.V(5).InfoS("Adding image ID with runtime class to currentImages", "imageKey", imageKey, "runtimeHandler", image.Spec.RuntimeHandler)
+		}
```

Comment:
lower in this file the log messages just include runtimeHandler and leave it "" if not enabled.  Since we are doing that below should we follow the same pattern here?

```suggestion
		if isRuntimeClassInImageCriAPIEnabled {
			imageKey = getImageTuple(image.ID, image.Spec.RuntimeHandler)
		}
		
		klog.V(5).InfoS("Adding image ID to currentImages", "imageID", imageKey, "runtimeHandler", image.Spec.RuntimeHandler)
```

Reply:
![image](https://github.com/kubernetes/kubernetes/assets/99994218/adbe3a18-faf4-49c3-945b-655b158cfa0d)

Wanted to explicitly call out the difference between imageID and imageKey when featuregate is enabled atleast for alpha phase. That's why I left it unchanged